tests/test_io/test_renderers.py

```diff
@@ -303,7 +303,7 @@ def test_repr_html(renderer):
         "window.PlotlyConfig = {MathJaxConfig: 'local'};</script>\n        "
         '<script charset="utf-8" src="'
         + plotly_cdn_url()
-        + '"></script>                '
+        + '" integrity="sha256-oy6Be7Eh6eiQFs5M7oXuPxxm9qbJXEtTpfSI93dW16Q=" crossorigin="anonymous"></script>                '
```

Comment:
Not really sure what this is testing but shouldn't this use a generated hash ?

Reply:
Ah, good catch! I think this would have broken the test with the next release. Fixed.